easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,422 @@
+    }),
+    ('distlib', '0.3.4', {
+        'source_tmpl': 'distlib-%(version)s.zip',
+        'checksums': ['e4b58818180336dc9c529bfb9a0b58728ffc09ad92027a3f30b7cd91e3458579'],
+    }),
+    ('filelock', '3.2.1', {
+        'checksums': ['9cdd29c411ab196cf4c35a1da684f7b9da723696cb356efa45bf5eb1ff313ee3'],
+    }),
+    ('importlib_resources', '3.3.1', {
+        'checksums': ['0ed250dbd291947d1a298e89f39afcc477d5a6624770503034b72588601bcc05'],
+    }),
+    ('platformdirs', '2.0.2', {
+        'checksums': ['3b00d081227d9037bbbca521a5787796b5ef5000faea1e43fd76f1d44b06fcfa'],
+    }),
+    ('virtualenv', '20.13.0', {
+        'checksums': ['d8458cf8d59d0ea495ad9b34c2599487f8a7772d796f9910858376d1600dd2dd'],
+    }),
+    ('docopt', '0.6.2', {
+        'checksums': ['49b3a825280bd66b3aa83585ef59c4a8c82f2c8a522dbe754a8bc8d08c85c491'],
+    }),
+    ('joblib', '0.14.1', {
+        'checksums': ['0630eea4f5664c463f23fbf5dcfc54a2bc6168902719fa8e19daf033022786c8'],
+    }),
+    ('chardet', '4.0.0', {
+        'checksums': ['0d6f53a15db4120f2b08c94f11e7d93d2c911ee118b6b30a04ec3ee8310179fa'],
+    }),
+    ('certifi', '2021.10.8', {
+        'checksums': ['78884e7c1d4b00ce3cea67b44566851c4343c120abd683433ce934a68ea58872'],
+    }),
+    ('urllib3', '1.26.8', {
+        'checksums': ['0e7c33d9a63e7ddfcb86780aac87befc2fbddf46c58dbb487e0855f7ceec283c'],
+    }),
+    ('requests', '2.27.1', {
+        'checksums': ['68d7c56fd5a8999887728ef304a6d12edc7be74f1cfa47714fc8b414525c9a61'],
+    }),
+    ('xlrd', '2.0.1', {
+        'checksums': ['f72f148f54442c6b056bf931dbc34f986fd0c3b0b6b5a58d013c9aef274d0c88'],
+    }),
+    ('py_expression_eval', '0.3.14', {
+        'checksums': ['ea60f9404a18346d5a63854db21c50666dfb4274ae111000165b31c6f8ab93f1'],
+    }),
+    ('tabulate', '0.8.9', {
+        'checksums': ['eb1d13f25760052e8931f2ef80aaf6045a6cceb47514db8beab24cded16f13a7'],
+    }),
+    ('ujson', '2.0.3', {
+        'checksums': ['bd2deffc983827510e5145fb66e4cc0f577480c62fe0b4882139f8f7d27ae9a3'],
+    }),
+    ('atomicwrites', '1.4.0', {
+        'checksums': ['ae70396ad1a434f9c7046fd2dd196fc04b12f9e91ffb859164193be8b6168a7a'],
+    }),
+    ('py', '1.11.0', {
+        'checksums': ['51c75c4126074b472f746a24399ad32f6053d1b34b68d2fa41e558e6f4a98719'],
+    }),
+    ('scandir', '1.10.0', {
+        'checksums': ['4d4631f6062e658e9007ab3149a9b914f3548cb38bfb021c64f39a025ce578ae'],
+    }),
+    ('pathlib2', '2.3.6', {
+        'checksums': ['7d8bcb5555003cdf4a8d2872c538faa3a0f5d20630cb360e518ca3b981795e5f'],
+    }),
+    ('zipp', '1.2.0', {
+        'checksums': ['c70410551488251b0fee67b460fb9a536af8d6f9f008ad10ac51f615b6a521b1'],
+    }),
+    ('configparser', '4.0.2', {
+        'checksums': ['c7d282687a5308319bf3d2e7706e575c635b0a470342641c93bea0ea3b5331df'],
+    }),
+    ('contextlib2', '0.6.0.post1', {
+        'checksums': ['01f490098c18b19d2bd5bb5dc445b2054d2fa97f09a4280ba2c5f3c394c8162e'],
+    }),
+    ('importlib_metadata', '1.7.0', {
```

Comment:
according to pypi 2.1.2 is the latest for py2.7

Reply:
Here too, an older `importlib_metadata` is required for one of the extensions below

Reply:
ignore this one, poetry-core requires importlib-metadata <2